addon/components/sortable-group.js

```diff
@@ -18,29 +19,35 @@ export default Component.extend({
     @property items
     @type Ember.NativeArray
   */
-  items: computed(() => { return a(); }),
+  items: computed({
+    get() { return a(); }
```

Comment:
This change should not be necessary.

`computed(function() {})` is just syntax sugar to `computed({ get() {} })`, and shouldn't throw any deprecation warning.